config/example_ids_configuration.ini

```diff
@@ -0,0 +1,253 @@
+﻿[Versions]
+libueye_api.so=4.94.1557
+ueyed=4.94.2.0
+
+
+[Sensor]
+Sensor=UI524xCP-C
```

Comment:
I am a little hesitant about an example ids configuration just because of how specific it is and its lack of generality compared to the ros configuration. The ros configuration will work for most IDS cameras and any ueye api > 4.94, but the specificity of this ids config makes me question its usefulness a little. Thoughts?

Reply:
I was also having a similar thought upon seeing the old .ini + launcher. I figured it might have been used for a maintainer who is using them do so some quick, manual QA before making a release on the camera configuration they happen to have. It's very possible it won't work for anyone else. Aside from that, it's a useful thing in the readme to be able to link to so someone can see what it's all about without having to create one.

Without having completely committed to anything yet in this PR, I was thinking along the lines of:

* An example .ini file just to be able to link to from the README for the reasons stated above
* A section in the README stepping someone through the process of creating and loading the .ini in a launch file (there's already a TODO for that there)

How does that sound?

Reply:
I think that sounds good. I definitely like the README having the documentation there. 

It already has `example` in the prefix, so I don't think a rename is really necessary.